### PR TITLE
feat(notifier): adds source to channels

### PR DESCRIPTION
**Pull request by jurajpiar**
Resolves https://rsklabs.atlassian.net/browse/RMKT-849

Depends on https://github.com/rsksmart/rif-marketplace-cache/pull/787
Screenshot 2021-08-08 at 02 59 10

 
https://user-images.githubusercontent.com/36888576/128617481-906b493a-de28-44df-861f-2299bfeed93a.mov

### CODE REVIEW

File: src/constants/notifier/strings.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { SupportedEventType, SupportedEventChannel } from 'config/notifier'
 
 export const notifierChannelPlaceHolder: Record<SupportedEventChannel, string> = {
   API: 'Enter api destination',
-  EMAIL: 'Enter email address',
+  EMAIL: 'Enter email address destination',
```
Comment (vjgene) [inline src/constants/notifier/strings.ts]:
```suggestion
  EMAIL: 'Enter email destination',
```

File: src/components/organisms/notifier/NotificationChannelCreate.tsx [hunk 2]
```diff
@@ -65,55 +75,66 @@ const NotificationChannelCreate: FC<Props> = ({
...
+          <FormControl variant="outlined" className={classes.formControl}>
+            <Select
+              className={classes.select}
+              id="channel-select"
+              value={notifierChannel.type}
+              onChange={handleChannelChange}
+            >
+              {
+        availableChannels.map(
+          ({ type }) => <MenuItem value={type}>{type}</MenuItem>,
```
Comment (itofarina) [inline src/components/organisms/notifier/NotificationChannelCreate.tsx]:
```suggestion
          ({ type }) => <MenuItem key={type} value={type}>{type}</MenuItem>,
```

File: src/components/organisms/notifier/details/utils.tsx [hunk 3]
```diff
@@ -35,14 +36,26 @@ export const eventDisplayItemIterator = ({
...
-      {Array.isArray(notificationPreferences)
-        ? notificationPreferences.join(', ')
-        : notificationPreferences}
+      <Grid container direction="column">
+        {cleanPrefs.map((channelType) => {
+          const channel = channels.find(
+            ({ type }) => type === channelType,
+          ) as EventChannel
+          return (
+            <Grid item>
```
Comment (itofarina) [inline src/components/organisms/notifier/details/utils.tsx]:
```suggestion
            <Grid item key={channelType}>
```

**Review by vjgene — APPROVED** [review]
Looks good!

**Review by itofarina — APPROVED** [review]
Nice one!

**Comment by jurajpiar** [comment]
- [ ] **PLEASE, DO NOT MERGE UNTIL THE RELATED CACHE PR IS MERGED AND DEPLOYED!**

**Review by 0xartem — APPROVED** [review]
nice!

**Pull request merged**